newdle/client/src/components/ParticipantTable.js

```diff
@@ -63,11 +69,12 @@ function AvailabilityRow({availability: {startDt, available}, setActiveDate, act
   const numberOfParticipants = useSelector(getNumberOfParticipants);
   const duration = useSelector(getNewdleDuration);
   const startTime = toMoment(startDt, 'YYYY-MM-DDTHH:mm');
+  const dispatch = useDispatch();
 
   return (
     <Table.Row
       className={styles['participant-row']}
-      onClick={() => setActiveDate(startDt)}
+      onClick={() => dispatch(setActiveDate(startDt))}
```

Comment:
Is it something that should be here actually? I think that `ParticipantTable` shouldn't be using `dispatch`. You can just pass a callback here that uses `dispatch` internally.